libs/langchain/pyproject.toml

```diff
@@ -395,7 +398,7 @@ build-backend = "poetry.core.masonry.api"
 #
 # https://github.com/tophat/syrupy
 # --snapshot-warn-unused    Prints a warning on unused snapshots rather than fail the test suite.
-addopts = "--strict-markers --strict-config --durations=5 --snapshot-warn-unused"
+addopts = "--strict-markers --strict-config --durations=5"
```

Comment:
why change this?

Reply:
Hi! My pytest doesn't support this argument so I removed it during development process. `poetry run pytest` did work. I will add this back :)